tests/infra/network.py

```diff
@@ -286,7 +286,7 @@ def _start_all_nodes(
                         )
                         self.wait_for_state(
                             node,
-                            "partOfPublicNetwork",
+                            "PartOfPublicNetwork",
```

Comment:
While we're here: we should make this a Python enum defining all of the possible values, rather than a string comparison.